third_party/silabs/silabs_executable.gni

```diff
@@ -61,14 +61,16 @@ template("silabs_executable") {
     flashing_image_name = output_base_name + ".rps"
   }
 
-  flashing_runtime_target = target_name + ".flashing_runtime"
   flashing_script_inputs = [
     "${chip_root}/scripts/flashing/silabs_firmware_utils.py",
     "${chip_root}/scripts/flashing/firmware_utils.py",
   ]
-  copy(flashing_runtime_target) {
-    sources = flashing_script_inputs
-    outputs = [ "${root_out_dir}/{{source_file_part}}" ]
+  if (defined(invoker.copy_py_files) && invoker.copy_py_files) {  #++++ added this
```

Comment:
The "copy" action only needs to be done once per build, however now that `silabs_executable` is being called many times it ends up trying to copy these same python files repeatedly which causes an error.  I could not find a clean way of making it only copy once, so as a workaround I use a variable `copy_py_files` that controls whether the files get copied or not.  This variable is set to false after silabs_executable is invoked so the files won't be copied next time.